bugshaker/src/main/java/com/github/stkent/bugshaker/email/FeedbackEmailFlowManager.java

```diff
@@ -70,21 +80,32 @@ public void onClick(final DialogInterface dialog, final int which) {
             }
 
             if (shouldAttemptToCaptureScreenshot(activity)) {
-                if (environmentCapabilitiesProvider.canSendEmailsWithAttachments()) {
-                    try {
-                        sendEmailWithScreenshot(activity);
-                    } catch (final IOException exception) {
-                        final String errorString = "Screenshot capture failed";
-
-                        toaster.toast(errorString);
-
-                        Logger.e(errorString);
-                        Logger.printStackTrace(exception);
-
-                        sendEmailWithNoScreenshot(activity);
-                    }
+                if (emailCapabilitiesProvider.canSendEmailsWithAttachments()) {
+                    screenshotProvider.getScreenshotUri(activity)
+                            .observeOn(AndroidSchedulers.mainThread())
```

Comment:
Add a `first()` or similar here to be explicit that we only expect to process a single result?